cypress/integration/01 - signup-login.ts

```diff
@@ -3,15 +3,15 @@ describe("Signup/Login", () => {
         cy.signup();
     });
 
-    // it("can login without errors", () => {
-    //     cy.login();
-    // });
+    it("can login without errors", () => {
+        cy.login();
+    });
 
-    // it("can unlock without errors", () => {
-    //     cy.login();
+    it("can lock/unlock without errors", () => {
+        cy.login();
 
-    //     cy.lock();
+        cy.lock();
 
-    //     cy.unlock();
-    // });
+        cy.unlock();
+    });
 });
```

Comment:
Just for my understanding: These tests each depend on a specific state left by the previous "command" and need to run in sequence, correct? Like the login test requires the signup test to complete first so there is an existing account, right?